lnbits/wallets/breez.py

```diff
@@ -0,0 +1,177 @@
+                return StatusResponse(f"Failed to connect to breez, got: '{exc}...'", 0)
+
+            return StatusResponse(None, int(node_info.max_payable_msat))
+
+        async def create_invoice(
+            self,
+            amount: int,
+            memo: Optional[str] = None,
+            description_hash: Optional[bytes] = None,
+            unhashed_description: Optional[bytes] = None,
+            **kwargs,
+        ) -> InvoiceResponse:
+            # if description_hash or unhashed_description:
+            #     raise Unsupported("description_hash and unhashed_description")
+
+            breez_invoice: breez_sdk.ReceivePaymentResponse = (
+                self.sdk_services.receive_payment(
+                    breez_sdk.ReceivePaymentRequest(
+                        amount * 1000,  # breez uses msat
+                        memo,
+                        preimage=kwargs.get("preimage"),
+                        opening_fee_params=None,
+                    )
+                )
+            )
+
+            return InvoiceResponse(
+                True,
+                breez_invoice.ln_invoice.payment_hash,
+                breez_invoice.ln_invoice.bolt11,
+                None,
+            )
+
+        async def pay_invoice(
+            self, bolt11: str, fee_limit_msat: int
+        ) -> PaymentResponse:
+            invoice = lnbits_bolt11.decode(bolt11)
+
+            try:
+                req = breez_sdk.SendPaymentRequest(bolt11=bolt11)
+                payment: breez_sdk.Payment = self.sdk_services.send_payment(req)
+            except Exception as exc:
+                logger.info(exc)
+                try:
+                    # try to report issue to Breez to improve LSP routing
+                    self.sdk_services.report_issue(
+                        breez_sdk.ReportIssueRequest.PAYMENT_FAILURE(
+                            breez_sdk.ReportPaymentFailureDetails(invoice.payment_hash)
+                        )
+                    )
+                except Exception as ex:
+                    logger.info(ex)
+                # assume that payment failed?
+                return PaymentResponse(
+                    False, None, None, None, f"payment failed: {exc}"
+                )
+
+            assert not payment.pending, "payment is pending"
+            # let's use the payment_hash as the checking_id
+            checking_id = invoice.payment_hash
+
+            return PaymentResponse(
+                True,
+                checking_id,
+                payment.fee_msat,
+                payment.details.data.payment_preimage,
+                None,
+            )
+
+        async def get_invoice_status(self, checking_id: str) -> PaymentStatus:
+            invoice = self.sdk_services.payment_by_hash(checking_id)
+            if invoice is None:
+                return PaymentStatus(None)
+            assert invoice.payment_type == breez_sdk.PaymentType.RECEIVED
+            return PaymentStatus(True)
```

Comment:
You can check the payment status here
```suggestion
            payment = self.sdk_services.payment_by_hash(checking_id)
            if payment is None:
                return PaymentStatus(None)
            assert payment.payment_type == breez_sdk.PaymentType.RECEIVED
            return PaymentStatus(payment.status == breez_sdk.PaymentStatus.COMPLETE)
```